### ISSUE TITLE
Voice over is not announcing role as combo box and state as collapsed for "Project" combobox:A11y_.NET maui_user can creat a tak_Screen reader

### ISSUE THREAD

**Issue by kapilvaishna** [dotnet/maui#30899]
**Please do not close this bug. This bug should only be closed by Trusted Tester after verification.**
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## GitHub Tags
#A11yTCS; #A11yMAS; #WCAG4.1.2; #ScreeReader; #BM_.NETMaui_Mac_JUL2025; #DesktopApp; #Device; #FTP; #A11ySev1; #.NET MAUI SDKs; #Mac; #VoiceOver;

## Environment Details:
Application: Developer Balance (.NET Maui)
MAC
M2

## Repro Steps:

1. Turn on Voiceover
2. Install and open the Developer Balance app
3. Press Control + Option + Right arrow key to navigate "Add task" button and press  Control + Option + space key.
4. Press Control + Option + Right arrow key to navigate "Project" combo box and press  Control + Option + space key to expand.
5. Observe the issue 

## Actual Result:
When navigate to "Project" combo box using Control + Option + Right arrow, voice over is not announcing the role as combo box and state as collapsed.

**Note:**
Same issue repro with "Category" combo box present in create a project page

## Expected Result:
When navigate to "Project" combo box using Control + Option + Right arrow voice hover should announce role as combo box , name as project and state as collapsed.

## User Impact:
Screen user will not get the correct control type and state of the control. 

## Attachment

https://github.com/user-attachments/assets/cb3e2a0c-38be-49f9-a7ba-3d41924001f1

### Similar issue

https://github.com/user-attachments/assets/242f25f8-0462-430f-aa2b-b24b507837a7

**Comment by ghost** [comment]
This issue has been verified in Visual Studio Code 1.102.2 with MAUI versions 9.0.0 and 9.0.90. It can be reproduced on MacCatalyst platform.

**Comment by SuthiYuvaraj** [comment]
Hi @kapilvaishna , In the `Developer Balance` app, on the Task Details page, you're currently using a combination of `SfTextInputLayout` and `Picker`, which results in a ComboBox-like control. As a result, VoiceOver reads it as a "Text Edit" field.

To improve accessibility and clearly convey the control's role and state, you can use the following SemanticProperties:

```
 <sf:SfTextInputLayout
                    IsVisible="{Binding IsExistingProject}"
                    SemanticProperties.Description="ComboBox to select project"
                    SemanticProperties.Hint="State Collapsed"
                    Hint="Project">
 <Picker
          ...
   SemanticProperties.Description="Project"
    SemanticProperties.Hint="Which project this task belongs to"/>
 </sf:SfTextInputLayout>
```
Here I have attached the output video for your reference:

https://github.com/user-attachments/assets/0cd4be29-d287-4f40-9f03-a6094b2f2826

**Comment by jsuarezruiz** [comment]
@sheiksyedm I noticed this has been reopened, could you share the current status? Let me know if there’s any way I can help.

**Comment by kapilvaishna** [comment]
@SuthiYuvaraj In your previous comment you mentioned to use SemanticProperties. could you please help how can i do that. because this issue is reproducing for me in version 8.

Also as issue is not reproducing for you. Could you please share the two recordings with voice hover where you navigate to project and category combo box, than expand combo box list and than navigate to some list item and select that list item. So i can close this bug also https://github.com/dotnet/maui/issues/30897
Please share MAC version and application version as well.

**Comment by SuthiYuvaraj** [comment]
@kapilvaishna This issue has already been resolved in DeveloperBalance, so no further changes are required. I have verified it on my end and could not reproduce the issue. Please find the reference video and tested macOS details attached for your reference.

**Tested Mac Details:**
Chip: Apple M3 Max
VS Code version: 1.104.1

**Output video**

https://github.com/user-attachments/assets/6e1e04ea-215f-4fd3-8c56-23380186a02b

**Comment by kapilvaishna** [comment]
@SuthiYuvaraj issue is partially fixed voice hover is still not announcing the collapsed state for combo box.

Developer balance version 10 (10)
Chip Apple M2

https://github.com/user-attachments/assets/4f933f28-c062-424d-b32f-a669c5feef53

**Comment by Yash14j** [comment]
GitHub Tags:
#External;#ThirdParty;#External:SyncfusionToolkitChart;#ResolvedbyTT;

**Comment by kubaflo** [comment]
Hi! @kapilvaishna I think it works correctly. Could you please retest and close it everything works. Thanks!

https://github.com/user-attachments/assets/0260fd16-f6ac-4144-90f3-9543b106174d

**Comment by kapilvaishna** [comment]
@kubaflo Now Screen reader is announcing the role as combo box but still screen reader is not announcing any thing about expanded stats or expanded list.

https://github.com/user-attachments/assets/0509f5fe-156d-428f-ba60-2e06f3b788e9

**Comment by SuthiYuvaraj** [comment]
Hi @kapilvaishna , @kubaflo , The issue has been reproduced on .NET 11, and it appears to be related to the CommunityToolkit. I have logged an issue in the CommunityToolkit repository for further investigation. Kindly refer to that (#[1166](https://github.com/CommunityToolkit/dotnet/issues/1166)) issue for further updates and reference.

**Comment by kubaflo** [comment]
@SuthiYuvaraj Thanks!

**Comment by kapilvaishna** [comment]
Closing this bug and logged new bug https://github.com/dotnet/maui/issues/36471
